.github/workflows/ui-rm-head-2.7-matrix.yaml

```diff
@@ -14,11 +14,11 @@ on:
         type: boolean
       k8s_downstream_version:
         description: Rancher cluster downstream version to use
-        default: '"v1.27.13+k3s1"'
+        default: '"v1.30.4+k3s1"'
         type: string
       k8s_upstream_version:
         description: Rancher cluster upstream version to use
-        default: '"v1.27.13+k3s1"'
+        default: '"v1.30.4+k3s1"'
```

Comment:
In rancher 2.7-head, there is nothing higher than 1.27 for the kubernetes downstream cluster:
![image](https://github.com/user-attachments/assets/0f39e50a-2a95-494a-926b-e87dc8b653e9)

Reply:
Yes I saw it, fixed!